modules/programs/neovim.nix

```diff
@@ -121,6 +133,15 @@ in {
         '';
       };
 
+      generatedConfigViml = mkOption {
```

Comment:
why move this ?

Reply:
This got accidentally moved when it got readded after deletion.